teamA/jeongDoYoung/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = yourPortNumber
```

dbmate 연결 목적의 DATABASE_URL을 추가해주세요~!

개인이 로컬 환경에서 어떠한 rdbms를 쓰는지 그 힌트가 될 수 있는 단어와, 포트번호를 지우고 중의적인 단어를 대신 사용해주세요.

```suggestion
DATABASE_URL="rdbmsType://username:password@iphost:portNumber/databaseName"

PORT = yourPortNumber
```